File: CursoLogica/Linguagens/Python/Atividades/soma_impares.py
print("Digite dois números: ")
valor1 = int(input())
valor2 = int(input())

if valor1 > valor2:
    troca = valor2
    valor2 = valor1
    valor1 = troca

soma = 0
for i in range(valor1+1, valor2-1, 2):
    if i % 2 == 0:
        i += 1
    soma = soma + i
print(f"SOMA DOS ÍMPARES = {soma}")

File: CursoLogica/Linguagens/C/Atividades/soma_impares.c
#include <stdio.h>

int main() {

    int valor1, valor2, troca, soma, i;

    printf("Digite dois numeros:\n");
    scanf("%d", &valor1);
    scanf("%d", &valor2);

    if (valor1 > valor2) {
        troca = valor1;
        valor1 = valor2;
        valor2 = troca;
    }

    soma = 0;
    for (i = valor1+1; i < valor2; i++) {
        if (i % 2 != 0) {
            soma = soma + i;
        }
    }
    printf("SOMA DOS IMPARES = %d", soma);

    return 0;
}

